### PR TITLE
French branch

### PR DESCRIPTION
**Pull request by alexture**
Not sure if I'm supposed to do this? Also, it doesn't include the changes from my English version pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,167 +1,168 @@
...
+- [Espagnol](https://web.archive.org/web/20220511155347/https://github.com/JoseDeFreitas/awesome-translations/tree/es_ES)
+- [Italien](https://github.com/niedev/awesome-translations/tree/it_IT)
+- [Français](https://github.com/alexture/awesome-translations/tree/fr_FR)
+
+## Plateformes
 
-- [English](https://github.com/mbiesiad/awesome-translations)
-- [Polish](https://github.com/mbiesiad/awesome-translations/tree/pl_PL)
-- [Spanish](https://web.archive.org/web/20220511155347/https://github.com/JoseDeFreitas/awesome-translations/tree/es_ES)
-- [Italian](https://github.com/niedev/awesome-translations/tree/it_IT)
```
Comment (mbiesiad) [inline README.md:24]: One of the solution could be to add as 25 line - [French](https://github.com/alexture/awesome-translations/blob/fr_FR/README.md) and undo other changes in this file (as main file, should be in English).

Comment (mbiesiad) [reply README.md:24]: `- [French](https://github.com/alexture/awesome-translations/blob/fr_FR/README.md)`

**Review by mbiesiad — CHANGES_REQUESTED** [review]
Changes needed

**Comment by alexture** [comment]
Ohhh got it, sorry for getting this wrong!

**Pull request closed without merging**